### PR TITLE
Fix/1931/16.12

### PR DESCRIPTION
**Pull request by landryb**
Cherrypicked from #1936

### CODE REVIEW

File: console/pom.xml [hunk 1]
```diff
@@ -191,7 +191,246 @@
   </build>
 
   <profiles>
-
```
Comment (pmauduit) [inline console/pom.xml]: Perplexed about this modification

Comment (landryb) [reply console/pom.xml]: was removed in da4f0ff :)

**Comment by landryb** [comment]
`make deb-build-georchestra` in the 16.12 branch with the last commits generate the following debs:
```
$find . -path '*target*' -name '*.deb'
./atlas/target/georchestra-atlas_16.12-20180614-2008~7cd1346_all.deb
./ldapadmin/target/georchestra-ldapadmin_16.12-20180614-2008~7cd1346_all.deb
./geowebcache-webapp/target/georchestra-geowebcache_16.12-20180614-2008~7cd1346_all.deb
./header/target/georchestra-header_16.12-20180614-2008~7cd1346_all.deb
./mapfishapp/target/georchestra-mapfishapp_16.12-20180614-2008~7cd1346_all.deb
./catalogapp/target/georchestra-catalogapp_16.12-20180614-2008~7cd1346_all.deb
./extractorapp/target/georchestra-extractorapp_16.12-20180614-2008~7cd1346_all.deb
./downloadform/target/georchestra-downloadform_16.12-20180614-2008~7cd1346_all.deb
./cas-server-webapp/target/georchestra-cas_16.12-20180614-2008~7cd1346_all.deb
./analytics/target/georchestra-analytics_16.12-20180614-2008~7cd1346_all.deb
./geoserver/webapp/target/georchestra-geoserver_16.12-20180614-2008~7cd1346_all.deb
./security-proxy/target/georchestra-security-proxy_16.12-20180614-2008~7cd1346_all.deb
./geonetwork/web/target/georchestra-geonetwork3_3.0.4+201806142218-20180614-2008~7cd1346_all.deb

```

note the specificity for geonetwork... *shrug*

**Comment by landryb** [comment]
Can this one be merged to 16.12 and 17.12 now that georchestra/geonetwork#72 and  georchestra/geonetwork#71 have been merged ? They depend on this one for packageVersion to be set.. i think. Too many branches, too much git.

**Pull request merged**